simplemachines.xml

```diff
@@ -0,0 +1,19 @@
+<ruleset name="Simple Machines">
+	<target host="simplemachines.org"/>
+	<target host="www.simplemachines.org"/>
+	<target host="adsystem.simplemachines.org"/>
+	<target host="custom.simplemachines.org"/>
+	<target host="dev.simplemachines.org"/>
+	<target host="download.simplemachines.org"/>
+	<target host="wiki.simplemachines.org"/>
+	<target host="support.simplemachines.org"/>
+	<target host="media.simplemachinesweb.com"/>
+	<test url="http://custom.simplemachines.org/"/>
+	<test url="http://dev.simplemachines.org/"/>
+	<test url="http://download.simplemachines.org/"/>
+	<test url="http://dev.simplemachines.org/"/>
+	<test url="http://wiki.simplemachines.org/"/>
+	<test url="http://support.simplemachines.org/"/>

```

Comment:
You don't need to add test urls as long as they are the same as the listed targets.